vmf/scripts/mods/vmf/modules/ui/options/vmf_options_view.lua

```diff
@@ -1069,32 +1134,34 @@ local function create_header_widget(widget_definition, scenegraph_id)
 
       fav_icon = {
         size = {30, 30},
-        offset = {15, offset_y + 25, 3},
+        color = { 125, 255, 255, 255 },
+        offset = {widget_size[1] - 45, offset_y + 25, 3},
         masked = true
       },
 
       fav_arrow_up = {
         size = {20, 20},
-        offset = {20, offset_y + 57, 3},
+        offset = {widget_size[1] - 40, offset_y + 57, 3},
         color = {90, 255, 255, 255},
         masked = true
       },
 
       fav_arrow_down = {
         size = {20, 20},
-        offset = {20, offset_y + 3, 3},
+        offset = {widget_size[1] - 40, offset_y + 3, 3},
         angle = math.pi,
         pivot = {10, 10},
         color = {90, 255, 255, 255},
         masked = true
       },
 
       text = {
-        offset = {60, offset_y + 18, 3},
-        font_size = 28,
+        offset = {60, offset_y + widget_size[2] / 2, 3},
+        font_size = 24,
```

Comment:
24 font is a side effect of making it smaller, then forgetting original. I can change it back if you want